config/samples/feature.streaming.clicks.yaml

```diff
@@ -12,7 +12,7 @@ spec:
   primitive: int
   freshness: 10s
   staleness: 1m
-  connector:
+  source:
```

Comment:
Should this be `data_source` for clarity?

Reply:
with k9s naming convention, it should be `dataSource`. Are we okay with that?